test_data/TEST_PRO/readme.txt

```diff
@@ -0,0 +1,9 @@
+$ python3 ../script/validate.py
```

Comment:
Could add arguments field for directories containing TSVs or for individual TSVs